### PR TITLE
add dockerfile

### PR DESCRIPTION
**Pull request by Byczax**
Signed-off-by: Byczax <byczqq136@gmail.com>

### CODE REVIEW

**Review by sdatko — CHANGES_REQUESTED** [review]
Please rebase to one commit :-)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+run apt-get update 
+run apt-get install git python3 python3-pip -y
+run apt-get clean
+
+run pip3 install git+https://github.com/informatyzacja-sspwr-projekty/CertMinder.git
+
+copy certminder.yml /certminder.yml
+
+cmd certminder -s -c
```
Comment (karolba) [inline Dockerfile]: change `run` -> `RUN`, `from` -> `FROM`, ... to comply with common Docker standards

**Review by karolba — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (sdatko) [reply Dockerfile]: Yes, please!

**Review by karolba — APPROVED** [review]

**Review by sdatko — APPROVED** [review]
Good enough

**Pull request merged**